EnglishFilter/sections/antiadblock.txt

```diff
@@ -781,6 +781,8 @@ btcbunch.com#%#//scriptlet("abort-on-property-write", "TestAd")
 @@||googletagmanager.com/gtag/js$domain=btcbunch.com
 @@||static.surfe.pro/js/net.js$domain=btcbunch.com
 !#endif
+! https://github.com/AdguardTeam/AdguardFilters/issues/115641
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js
```

Comment:
`||claimfey.com/cryptonews/wp-content/uploads/bqanwcfp.js`
You should have found the initiator script.